### ISSUE TITLE
Front end implementation of "Project" CRUD

### ISSUE THREAD

**Issue by tanerochris** [BuildForSDG/voxnostra#15]
Front implementation of creating , Reading, Updating and Deleting of a Project.
Add Project
Update Project
View project
List projects

**Comment by ndounalex** [comment]
Goog morning, the mockups for these interfaces already been done?

**Comment by tanerochris** [comment]
no, you can do the minimal design for now, and implement the front end, styling can be added later

**Comment by ndounalex** [comment]
who must set up the application layout?
I think we can use a free administration template and use it as a layout. 
we could use this for example.
![image](https://user-images.githubusercontent.com/15440117/81403546-e33ce580-9133-11ea-8195-5b74a2495682.png)

**Comment by tanerochris** [comment]
hello @ndounalex the design is already available ,let's work with that, check in the docs under mockups of @cliffordten  PR #17

**Comment by tanerochris** [comment]
Any updates on this task?
